src/test-helpers.tsx

```diff
@@ -88,6 +90,15 @@ export const fakeVersion: ExternalVersion = Object.freeze({
   version: '1.0',
 });
 
+export const testVersionFile: VersionFile = Object.freeze({
```

Comment:
Note: I know this is ugly/confusing. The problem is that we now have three different "types" of VersionFile. 

1. There is [`ExternalVersionFile`](https://github.com/mozilla/addons-code-manager/blob/1c2910b410402786a1986851106b27757ff59a82/src/reducers/versions.tsx#L39), which describes the structure of file information returned by the API. [`fakeVersionFile`](https://github.com/mozilla/addons-code-manager/blob/1c2910b410402786a1986851106b27757ff59a82/src/test-helpers.tsx#L36) in this `test-helpers.tsx` file represents an instance of that - the external representation of the data, which is what all of the `fakeXxx` objects represent.
2. There is [`InternalVersionFile`](https://github.com/mozilla/addons-code-manager/blob/1c2910b410402786a1986851106b27757ff59a82/src/reducers/versions.tsx#L85), which describes the internal representation of a file that we store in the redux store.
3. There is [`VersionFile`](https://github.com/mozilla/addons-code-manager/blob/1c2910b410402786a1986851106b27757ff59a82/src/reducers/versions.tsx#L92), which describes the structure of a file that the `versions` reducer returns to us. This is what the app uses when rendering anything. I introduced this awhile back and I was worried at the time that it was going to cause confusion, but it seemed to be the least evil way of doing it.

This means that we now really need to be able to have an object which is of type `VersionFile` for both unit tests and storybook, which is why I created this thing called `testVersionFile`. I didn't want to use `fakeXxx`, because our convention thus far has been that those represent external objects returned by the API.

I think a better solution would be to introduce a new function into the reducer for `createVersionFile`, which accepts data from both [`versionInfo`](https://github.com/mozilla/addons-code-manager/blob/1c2910b410402786a1986851106b27757ff59a82/src/reducers/versions.tsx#L169) and [`versionFiles`](https://github.com/mozilla/addons-code-manager/blob/1c2910b410402786a1986851106b27757ff59a82/src/reducers/versions.tsx#L172), and then constructs an object of type `VersionFile`. I started down that path, but felt like the patch was getting too big, so I figured maybe it was best to submit this for now, and do that refactor in a follow-up, if it is deemed valuable.

Reply:
Would https://github.com/mozilla/addons-server/issues/10891 fix those issues?

I have started to play with the Compare API and that's a mess, now `version.file` can have a `diff` instead of `content`, and the rest is unchanged (and not very useful). I believe we should fix the API responses now, because that will never scale.

Reply:
Yes, I think that https://github.com/mozilla/addons-server/issues/10891 coule potentially resolve this entire issue. I agree with bumping up its priority. I still think we can try to land things (especially as I now only have 2 days left before I leave for nearly 2 weeks), so maybe we can land this and live with it for awhile and fix it once https://github.com/mozilla/addons-server/issues/10891 lands?

Reply:
> so maybe we can land this and live with it for awhile and fix it once [mozilla/addons-server#10891](https://github.com/mozilla/addons-server/issues/10891) lands?

Yes!